### PR TITLE
(PA-3666) add tests to verify facts values from puppet

### PR DESCRIPTION
**Pull request by ciprianbadescu**

### CODE REVIEW

File: acceptance/tests/ensure_facter_values_for_external_overrides_custom_overrides_core_dotted_fact.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+      step 'create external and custom fact' do
+        create_remote_file(agent, File.join(foo_custom_facts_dir, "my_fizz_fact.rb"), <<-FILE)
+Facter.add('ruby.version') do
+  setcode do
+    '#{custom_ruby_value}'
+  end
+end
+FILE
+        create_remote_file(agent, File.join(foo_external_facts_dir, "my_fizz_fact.txt"),'ruby.version=#{external_ruby_value}')
```
Comment (gimmyxd) [inline acceptance/tests/ensure_facter_values_for_external_overrides_custom_overrides_core_dotted_fact.rb:31]: `'ruby.version=#{external_ruby_value}'` this needs double quotes or `external_ruby_value` won't get interpolated

**Comment by gimmyxd** [comment]
Jenkins please test this on windows2019-64a

**Review by gimmyxd — APPROVED** [review]

**Comment by GabrielNagy** [comment]
Jenkins please test this on windows2019-64a

**Comment by GabrielNagy** [comment]
Jenkins please test this on windows2019-64a

**Comment by ciprianbadescu** [comment]
Jenkins please test this on windows2019-64a

**Pull request merged**